test/fixtures/simple_example/main.tf

```diff
@@ -41,11 +41,11 @@ module "bastion" {
   project_id = var.project_id
   subnetwork = var.subnetwork
   zone       = data.google_compute_zones.main.names[0]
-  key_suffix = "_simple_example"
+  key_suffix = "_install_simple"
```

Comment:
This file can be removed as well, there is already a [text fixture](https://github.com/forseti-security/terraform-google-forseti/tree/master/test/fixtures/install_simple) and tests for the install_simple example.

Reply:
Looks like you have done that already, will mark this as resolved and wait for the build to complete.

Reply:
Already removed. I think this happened in the same minute. What a timing!

Reply:
Already removed.